docs/prereqs-ubuntu.md

```diff
@@ -38,6 +38,11 @@ sudo unzip protoc-${PROTOBUF_VERSION}-linux-x86_64.zip -d /usr/local
 sudo chmod +x /usr/local/bin/protoc
 ```
 
+## Dep
+
```

Comment:
did you run through these steps once? dep shell script doesn't just let you install without a GOPATH set

Reply:
Ok tested in a blank container. Fixed by adding paths, and little modification to path when building.